app/assets/javascripts/modules/host-content-iframe.js

```diff
@@ -0,0 +1,63 @@
+/* global Turbo */
+'use strict'
```

Comment:
Now that we're here, I have some questions on how we do JS here. This seems quite an old fashioned way of doing it with IIFEs and module patterns. Do we not use a bundler or something usually so we can use more modern features? If this is too big a discussion for this PR then we can take it offline.

Reply:
Yeah, this is something I have considered, but it's consistent with the overall GDS pattern used elsewhere and documented in the [Whitehall Javascript patterns](https://docs.publishing.service.gov.uk/repos/whitehall/javascript.html) and [GDS Javascript coding style](https://gds-way.digital.cabinet-office.gov.uk/manuals/programming-languages/js.html)

I've also had nothing but trouble bundling more modern Javascript in Rails (especially as part of the asset pipeline), so have stuck with the path of least resistance.